packages/gatsby/src/redux/actions.js

```diff
@@ -502,7 +503,7 @@ const typeOwners = {}
  *   }
  * })
  */
-actions.createNode = (
+const createNode = (
```

Comment:
NIT: it's a bit confusing to have `createNode` and `actions.createNode` be something else - does it make sense to make slight name change?

jsdoc being applied to this and not `actions.createNode` possibly can mess API reference page?

Reply:
@pieh hmm that was kind of on purpose
* didn't want to touch the action creator, just wrap it
* what should be documented is the thing that is passed down, which is the bound action creator, so we have to keep the name there; and we have to keep the name for what *is* passed down on the `actions` object
* action docs page looks ok
* that said, we *will* want to revisit this when we more selectively distinguish between public actions and internal actions.